### DISCUSSION TITLE
Algorithmic-Trading-Mobile

### DISCUSSION THREAD

**Discussion by cyberprophet** [Share-Invest/Algorithmic-Trading-Server#7] [Q&A]
![mobile](https://user-images.githubusercontent.com/48705422/216336738-62a01361-12ad-46c9-b6fd-98c61efa8f37.jpg)

**Comment by mrmind-manager** [comment]
![mobile](https://user-images.githubusercontent.com/45344929/216338019-fc69a7f8-c9d9-45f5-8462-5a3cf20e4252.jpg)

**Comment by mrmind-manager** [comment]
![SDI](https://user-images.githubusercontent.com/45344929/220412388-c4d3ae07-26b2-4e2a-8b2b-530752e30232.png)

**Comment by mrmind-manager** [comment]
![SDI](https://user-images.githubusercontent.com/45344929/220413520-176c9c8a-03f7-4d2a-9b2a-2c3d72a692b7.jpg)
